unity-renderer/Assets/Scripts/MainScripts/DCL/FatalErrorReporter/DefaultFatalErrorReporter.cs

```diff
@@ -8,7 +8,7 @@ public class DefaultFatalErrorReporter : IFatalErrorReporter
 
         public void Report(Exception exception)
```

Comment:
Is it ever referenced? Maybe remove it if no longer needed?

Reply:
Left it to discuss it in the daily. Its not referenced anywhere, so its probably safe to delete